modules/microshift-install-system-requirements.adoc

```diff
@@ -8,10 +8,10 @@
 
 The following conditions must be met prior to installing {microshift-short}:
 
-* A compatible version of {op-system} or {op-system-ostree}.
+* A compatible version of {op-system-base} or {op-system-ostree}.
 * AArch64 or x86_64 system architecture.
```

Comment:
The AArch64 looks funny.  Is there a reason the first two As are capitalized ?

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#aarch64